Toxygates/src/panomicon/ScalatraJSONServlet.scala

```diff
@@ -431,12 +433,18 @@ class ScalatraJSONServlet(scontext: ServletContext) extends ScalatraServlet
     val comment = params.get("comment").getOrElse("")
     val publicComment = params.get("publicComment").getOrElse("")
```

Comment:
Can you check that updating the publicComment here works? I started trying to test this PR, but I am unable to update the public comment. I put a println to make sure that I was correctly passing the parameter so it shouldn't be a frontend issue. Also, I don't think this should matter, but this was a batch update without a metadata file.

Reply:
This was indeed a bug, and the latest commits should fix that.

Reply:
It seems like both [3a48469](https://github.com/Toxygates/Panomicon/pull/1/commits/3a48469797be1dcd1e2dd8fc8fadd915f2d203ba) and [01982f3](https://github.com/Toxygates/Panomicon/pull/1/commits/01982f38cb844c9b027ebf71d251931763dba40b) have to do with this, but there's also a little more going on there?

Reply:
In hindsight, I think they should have been the same commit.
I committed 3a48649 after identifying a bug that I needed to fix (t:comment wasn't being updated). Then I ended up overhauling the same functionality even more in 01982f3, and I found a way to simplify what was being done by using updateSinglePredicate (which removes the single existing value for a predicate before adding a replacement value). This removed the need for the method "deleteBatchProperties".